provisioning/service/src/Auth/ProvisioningTokenCredential.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Threading;
+using Azure.Core;
+using Microsoft.Azure.Devices.Common.Service.Auth;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Auth
+{
+    internal class ProvisioningTokenCredential : IAuthorizationHeaderProvider
```

Comment:
Same about doc comments. See \iothub\service\src\DigitalTwin\Authentication\DigitalTwinTokenCredential.cs for example.